src/app/child-dev-project/children/children-list/list-relation/list-relation.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input, OnChanges, SimpleChanges } from "@angular/core";
+import { Child } from "../../model/child";
+import { ChildSchoolRelation } from "../../model/childSchoolRelation";
+import { ChildrenService } from "../../children.service";
+
+@Component({
+  selector: "app-list-relation",
+  templateUrl: "./list-relation.component.html",
+  styleUrls: ["./list-relation.component.scss"],
+})
+export class ListRelationComponent implements OnChanges {
+  @Input() id: string;
+  @Input() child: Child;
+  relation: ChildSchoolRelation;
+
+  constructor(private childrenService: ChildrenService) {}
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.hasOwnProperty("child")) {
```

Comment:
we gotta think and test performance of this approach. I would prefer to do loading once (for the complete child entity) and then have sub-components and columns work only as "display components" that get `@Input`s without using services to load data.

Did you have a particular reason to change and split this up?

Reply:
This was done to loose the coupling between `ChildrenListComponent` and `ChildrenService`. This would be a necessary step if we want to use the same component for the children and school (list-)overview.